src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -203,10 +201,6 @@
   <ItemGroup>
     <Content Include="Resources\Package.ico" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
I don't think you want to remove RequiredNetFX46FacadeAssemblies as these need to be in the VSIX.

Reply:
And I don't see you adding them back in. Are those making it in the VSIX thats used for Local F5 debugging (experimental instance)?

Reply:
Those are no longer needed now that nothing is being built against them.  I verified that F5-ing the experimental instance still works (and that the VSCode extension still works on Linux).